### DISCUSSION TITLE
Default Applications

### DISCUSSION THREAD

**Discussion by franzmondlichtmann** [labwc/labwc#929] [Q&A]
I would like to set a default File Manager on Labwc, but I did not find anything in the documentation.
Help would be appreciated

**Comment by Consolatis** [comment]
Labwc is just a boring compositor that renders windows and allows for moving them around.

It seems `xdg-mime query default inode/directory` will give you the current default.
As for how to set the defaults, there is some more information [here](https://wiki.archlinux.org/title/XDG_MIME_Applications) and possibly in other sources. Or there might be a menu for that within your file manager of choice.

**Comment by johanmalm** [comment]
Yes, I think you’d need a session manager or a small Desktop Environment for that.
I think that xfce and LXQt would support that. It would be cool to get them working on labwc.
